lib/keys.ts

```diff
@@ -22,6 +22,8 @@ export const UP = key('Up', 'up');
 export const BACK = key('Back', 'back');
 export const INSTANT_REPLAY = key('InstantReplay', 'instantReplay');
 export const INFO = key('Info', 'info');
+export const STAR = INFO;
+export const OPTIONS = INFO;
```

Comment:
We should make `star` and `options` show up as commands, e.g.

```js
client.command()
  .star()
  .options()
  .send()
```

They'll show up if you change them to:

```js
export const STAR = key('Info', 'star');
export const OPTIONS = key('Info', 'options');
```